clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -3980,6 +4005,10 @@ protected boolean isChunked() {
     return isChunked;
   }
 
+  void setRmdChunked(boolean isRmdChunked) {
```

Comment:
Nit:
This is for testing purpose, if yes, can you leave some javadoc here?

Reply:
Also, can we use other ways to test this code path? I think we can mock `Version` to simulate the RMD chunking, right? Having such setter is not good unless it is absolutely necessary.

Reply:
Good suggestion on using other ways to test. Removed this entirely.